app/Http/Controllers/BeatmapPacksController.php

```diff
@@ -86,6 +86,8 @@ public function index()
      * user_completion_data |
      *
      * @urlParam pack string required The tag of the beatmap pack to be returned.
+     *
+     * @queryParam legacy_only integer Whether or not to consider lazer scores for user completion data. Defaults to 1. Example: 0
```

Comment:
Doesn't it default to 0? I'm at least getting lazer scores without setting this param in the request and have to manually set it to 1 to exclude them.

Reply:
I think this should say it's also based on the user preference setting?

Reply:
user preference is not used for api request